packages/ui/form/PhoneInput.tsx

```diff
@@ -59,17 +58,13 @@ function BasePhoneInput({ name, className = "", onChange, ...rest }: PhoneInputP
 
 const useDefaultCountry = () => {
   const [defaultCountry, setDefaultCountry] = useState("us");
-  trpc.viewer.public.countryCode.useQuery(undefined, {
-    refetchOnWindowFocus: false,
-    refetchOnReconnect: false,
-    retry: false,
-    onSuccess: (data) => {
-      if (isSupportedCountry(data?.countryCode)) {
-        setDefaultCountry(data.countryCode.toLowerCase());
-      }
-    },
-  });
 
+  useEffect(() => {
+    const userCountryCode = navigator.language.split("-")[1];
+    if (userCountryCode && isSupportedCountry(userCountryCode)) {
+      setDefaultCountry(userCountryCode.toLowerCase());
+    }
+  }, []);
```

Comment:
The old way to grab the "defaultCountry" was always returning "us", so I changed how we can grab the user location, now we use the language selected on navigator to display the PhoneInput with this country code initially